webpack.config.js

```diff
@@ -26,7 +26,8 @@ module.exports = {
   },
   plugins: [
     new webpack.DefinePlugin({
-      SDK_VERSION: JSON.stringify(require('./package.json').version)
+      SDK_VERSION: JSON.stringify(require('./package.json').version),
+      STATE_TOKEN_COOKIE_NAME: JSON.stringify('oktaStateToken')
```

Comment:
instead of adding `STATE_TOKEN_COOKIE_NAME` to two different webpack builds, I've considered moving it to an `options` or `globals` module. What do you think?